tsconfig.json

```diff
@@ -35,8 +35,8 @@
     "src/**/*.vue",
     "tests/**/*.ts",
     "tests/**/*.tsx",
-    "src/store/index.js"
-, "tests/unit/components/PasswordInput.spec.js"  ],
+    "src/store/index.ts",
+    "tests/unit/components/PasswordInput.spec.ts"  ],
```

Comment:
I'm not sure if this one is necessary - wouldn't the `tests/**/*.ts` cover this one? 🤔

Reply:
Honestly I didn't add this manually, VSCode must have done that or other gnome 🤔